src/Products/Viewer/Controllers/ViewerApiController.cs

```
@@ -149,11 +150,19 @@ public HttpResponseMessage LoadDocumentDescription(PostedDataEntity postedData)
                 // return document description
                 return Request.CreateResponse(HttpStatusCode.OK, loadDocumentEntity);
             }
-            catch (System.Exception ex)
+            catch (PasswordProtectedFileException ex)
             {
                 // set exception message
                 return Request.CreateResponse(HttpStatusCode.Forbidden, new Resources().GenerateException(ex, password));
             }
+            catch (CorruptedOrDamagedFileException ex)
```

This catch can be removed since the next catch returns the same InternalServerError and handle all Exceptions except PasswordProtectedFileException exception.